lib/tests/relate.rs

```diff
@@ -207,3 +209,55 @@ async fn relate_with_complex_table() -> Result<(), Error> {
 	//
 	Ok(())
 }
+
+#[tokio::test]
+async fn schemafull_relate() -> Result<(), Error> {
+	let sql = r#"
+	INSERT INTO person [
+		{ id: 1 },
+		{ id: 2 }
+	];
+
+	DEFINE TABLE likes SCHEMAFULL;
```

Comment:
This should probably be type relation, and you can define the in and out fields inline if you prefer, but it doesn't change the result
```suggestion
	DEFINE TABLE likes SCHEMAFULL TYPE RELATION;
```